apps/backend/src/modules/projects/project.entity.ts

```diff
@@ -63,7 +63,7 @@ class ProjectEntity implements Entity {
 		};
 	}
 
-	public toObject(): ProjectGetAllItemResponseDto {
+	public toObject(): { description: string } & ProjectGetAllItemResponseDto {
```

Comment:
How about creating a separate type for a full project entity and use `Pick` for `ProjectGetAllItemResponseDto`?

Reply:
I think this is not a good solution, because ProjectGetAllItemResponseDto is used in other places where the description of the project is not needed

Reply:
I think we need to use ProjectGetByIdResponseDto we should already have it